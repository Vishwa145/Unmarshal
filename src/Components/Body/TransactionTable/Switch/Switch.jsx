import React from 'react';

function Switch (){
    return(
        <div className="SwitchTab">
            <button><h3>Ethereum</h3></button><button><h3>BSC</h3></button><button><h3>Ethereum</h3></button>
        </div>
    );
}

export default Switch;
