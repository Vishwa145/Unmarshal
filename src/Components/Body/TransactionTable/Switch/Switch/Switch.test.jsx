import React from "react";
import ReactDOM from "react-dom";
import Switch from "../Switch";
import renderer from "react-test-renderer"

it("render successfully", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Switch />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("Matches snapshot", ()=>{
    const tree = renderer.create(<Switch />).toJSON();
    expect(tree).toMatchSnapshot();
});