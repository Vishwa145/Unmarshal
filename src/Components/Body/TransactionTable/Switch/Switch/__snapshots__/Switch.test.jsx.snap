// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="SwitchTab"
>
  <button>
    <h3>
      Ethereum
    </h3>
  </button>
  <button>
    <h3>
      BSC
    </h3>
  </button>
  <button>
    <h3>
      Ethereum
    </h3>
  </button>
</div>
`;
